releasenotes/notes/grover-opt-primitive-de82d051d6cee2e4.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The :class:`~.GroverOptimizer` class has a new keyword argument, ``sampler`` which is
+    used to run the algorithm using an instance of the :class:`~.BaseSampler`
```

Comment:
> :class:`~.BaseSampler`

This needs to be a fully qualified path over to the BaseSampler in qiskit terra, otherwise its just highlighted in the output as it cannot be resolved like it is, since it's not part of optimization.

Reply:
Thanks. I update it.